core/generator.py

```diff
@@ -1,119 +1,125 @@
-                    for payload in payloads:
-                        vectors[10].add(payload.replace('<', '&lt;'))
+            handle_attribute_context(occurrence, breaker, attribute, vectors[6], vectors[10])
         elif context == 'comment':
-            lessBracketEfficiency = occurences[i]['score']['<']
-            greatBracketEfficiency = occurences[i]['score']['>']
-            breakerEfficiency = occurences[i]['score'][breaker]
-            ends = ['//']
-            if greatBracketEfficiency == 100:
-                ends.append('>')
-            if lessBracketEfficiency == breakerEfficiency == 100:
-                payloads = genGen(fillings, eFillings, lFillings, eventHandlers, tags, functions, ends, breaker, special)
-                for payload in payloads:
-                    vectors[10].add(payload)
-        elif context == 'script':
-            try:
-                script = scripts[index]
-            except IndexError:
-                try:
-                    script = scripts[0]
-                except:
-                    continue
-            closer = jsContexter(script)
-            validBreakers = ['\'', '"', '`']
-            scriptEfficiency = occurences[i]['score']['</scRipT/>']
-            greatBracketEfficiency = occurences[i]['score']['>']
-            breakerEfficiency = occurences[i]['score'][breaker]
-            ends = ['//']
-            if greatBracketEfficiency == 100:
-                ends.append('>')
-            if scriptEfficiency == 100:
-                breaker = r('</script/>')
-                payloads = genGen(fillings, eFillings, lFillings, eventHandlers, tags, functions, ends, breaker, special)
-                for payload in payloads:
-                    vectors[10].add(payload)
-            if closer:
-                suffix = '//\\'
-                if not breaker:
-                    closer = closer[1:]
-                if breakerEfficiency != 100:
-                    breaker = ''
-                for filling in jFillings:
-                    for function in functions:
-                        vector = breaker + closer + filling + function + suffix
-                        vectors[7].add(vector)
-            elif breakerEfficiency > 83:
-                suffix = '//'
-                for filling in jFillings:
-                    for function in functions:
-                        if '=' in function:
-                            function = '(' + function + ')'
-                        if breaker == '':
-                            filling = ''
-                        vector = '\\' + breaker + closer + filling + function + suffix
-                        vectors[6].add(vector)
+            handle_comment_context(occurrence, breaker, special, vectors[10])
+        elif context == 'script' and scripts:
+            script = index < len(scripts) scripts[index] else scripts[0]
```

Comment:
[Invalid Syntax]

Reply:
Not a python developer here, but looks like an inline if, or a trinity
```suggestion
            script = index < len(scripts) if scripts[index] else scripts[0]
```